### DISCUSSION TITLE
is justpy dead because people have migrated to NiceGUI?

### DISCUSSION THREAD

**Discussion by freemant2000** [elimintz/justpy#666] [General]
There have been few activities here...

**Comment by WolfgangFahl** [comment]
Indeed the Nicegui team is focussing on the modified approach of nicegui versus justpy. See https://github.com/zauberzeug/nicegui/issues/168.

When Eli allowed us to take the lead in this project we promised to make sure the tutorial examples still work and that's what we made sure so far. To progress we'd have to do major refactorings as nicegui already did and since nicegui is not pursuing the justpy compatability we have to find a course of actions ourselves here. 

Tentatively one way would be to look which of our examples are nicegui compatible and whether there is still some common ground between justpy and nicegui. Also it would be interesting to know how big the need is to keep justpy going for compatibility reasons.

**Comment by freemant2000** [comment]
But could NiceGUI be the refactoring that you have been looking for?
